src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -368,11 +375,7 @@ ClientRequest::process_op(
     co_return;
   }
 
-  DEBUGDPP("{}.{}: not completed, entering get_obc stage",
```

Comment:
nit: let's add "  `DEBUGDPP("{}.{}: not completed, entering check_already_complete_get_obc stage" ... )` instead.

Reply:
Um, we are actually going to enter "wait_scrub" phase here. I added the corresponding logs below.

Reply:
I meant adding log before entering `check_already_complete_get_obc`
DEBUGDPP("entering check_already_complete_get_obc stage" ... )
Shouldn't block just a suggestion :)

Reply:
Done:-)